vendor/wp-now/src/wp-now.ts

```diff
@@ -603,8 +603,7 @@ async function mountInternalMuPlugins( php: PHP, options: WPNowOptions ) {
 		`
 	);
 
-	// Only add the auto-updates disabling plugin for non-latest WordPress versions
-	if ( options?.wordPressVersion && options.wordPressVersion !== 'latest' ) {
+	if ( ! options.isWpAutoUpdating ) {
```

Comment:
Also good example of benefit from `isWpAutoUpdating` - the condition is much more simple.